tensorboard/plugins/mesh_visualizer/http_api.md

```diff
@@ -0,0 +1,65 @@
+# Mesh plugin HTTP API
+
+The mesh plugin name is `mesh`, so all its routes are under
+`/data/plugin/mesh`.
+
+## `/data/plugin/mesh/tags`
+
+Retrieves an index of tags containing mesh data.
+
+Returns a dictionary mapping from `runName` (quoted string) to
+dictionaries that map a `tagName` (quoted string) to an object
+containing that tag’s `displayName` and `description`, the latter of
+which is a string containing sanitized HTML to be rendered into the DOM.
+Here is an example:
+    {
+        "train_run": {
+            "mesh_color_tensor": {
+                "samples": 1
+            }, 
+            "point_cloud": {
+                "samples": 1
+            }
+        }
+    }
+
+Note that runs without any mesh tags are included as keys with value the empty dictionary.
```

Comment:
Please wrap consistently (preferably at 72, but 80 is also fine).

Reply:
I'll do 80 since my code is also 80.

Reply:
Okay, fine with me, though note that in Python code, text like
docstrings and comments _should_ be wrapped at 72 per PEP 8.